storage/innobase/xtrabackup/test/suites/rocksdb/log_status.sh

```diff
@@ -0,0 +1,21 @@
+#
+# PXB-2673 - Rocksdb backup fails if log_status_get is called more than one.
+#
+
+require_rocksdb
+require_debug_pxb_version
+
+start_server
+init_rocksdb
+
+mysql -e "INSTALL COMPONENT \"file://component_mysqlbackup\""
+mysql -e "CREATE TABLE t_logstatus (a INT AUTO_INCREMENT PRIMARY KEY) ENGINE=ROCKSDB" test
+mysql -e "INSERT INTO t_logstatus (a) VALUES (1), (2), (3)" test
+
+while true ; do
+    echo "INSERT INTO t_logstatus () VALUES ();"
+done | mysql test &
```

Comment:
how do we stop this insert loop (at the end of the test) that is running in background?

Reply:
Good catch, it has bitten me already in the past. Fixed.